tests/test_crypto/test_ledger_apis.py

```diff
@@ -0,0 +1,85 @@
+
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains the tests for the crypto/helpers module."""
+import logging
+import os
+import pytest
+
+from aea.crypto.ethereum import ETHEREUM, EthereumCrypto
+from aea.crypto.fetchai import FETCHAI, FetchAICrypto
+from aea.crypto.ledger_apis import LedgerApis, DEFAULT_FETCHAI_CONFIG
+from tests.conftest import CUR_PATH
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_ETHEREUM_CONFIG = ("https://ropsten.infura.io/v3/f00f7b3ba0e848ddbdc8941c527447fe", 3)
+fet_address = "B3t9pv4rYccWqCjeuoXsDoeXLiKxVAQh6Q3CLAiNZZQ2mtqF1"
+eth_address = "0x21795D753752ccC1AC728002D23Ba33cbF13b8b0"
+
+
+class TestLedgerApis:
+    """Test the ledger_apis module."""
+
+    def test_initialisation(self):
+        """Test the initialisation of the ledger APIs."""
+        ledger_apis = LedgerApis({ETHEREUM: DEFAULT_ETHEREUM_CONFIG})
+        assert ledger_apis.configs.get(ETHEREUM) == DEFAULT_ETHEREUM_CONFIG
+
+        unknown_config = ("UknownPath", 8080)
+        with pytest.raises(ValueError):
+            ledger_apis = LedgerApis({"UNKNOWN": unknown_config})
+
+    def test_token_balance(self):
```

Comment:
We need to mock the API calls here and below. Otherwise the tests depend on external services.

https://realpython.com/testing-third-party-apis-with-mocks/